### PR TITLE
[MODULAR] messes with maint-loot

### PR DESCRIPTION
**Pull request by RatFromTheJungle**
## About The Pull Request

revives the changes is #10480 , aswell as a maint-loot balance-pass

## How This Contributes To The Skyrat Roleplay Experience
there's been alot of really dumb shit in here for awhile. Also adds the stuff added in #10480, removes a556 as it's only really used in an unobtainable rifle, aswell as various other items from combat-clownshoes to a hypnogrenade that was in there for some reason
## Changelog

:cl:
add: Adds modsuit+ other-stuff to maint-loot
remove: removes alot of shit that has no business being in maint loot, from maintloot
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm [hunk 1]
```diff
@@ -19,7 +19,6 @@ GLOBAL_LIST_INIT(oddity_loot, list(//oddity: strange or crazy items
 	/obj/item/clothing/mask/animal/pig/cursed = 50,
 	/obj/item/clothing/mask/void_mask = 5,
 	/obj/item/clothing/shoes/bhop = 5,
-	/obj/item/clothing/shoes/clown_shoes/banana_shoes/combat = 5,
 	/obj/item/clothing/shoes/winterboots/ice_boots/ice_trail = 5,
```
Comment (tf-4) [inline modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm]:
```suggestion
```
Just as bad as clown shoes, megafauna loot

File: modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm [hunk 2]
```diff
@@ -19,7 +19,6 @@ GLOBAL_LIST_INIT(oddity_loot, list(//oddity: strange or crazy items
 	/obj/item/clothing/mask/animal/pig/cursed = 50,
 	/obj/item/clothing/mask/void_mask = 5,
```
Comment (tf-4) [inline modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm]:
```suggestion
```
This one is host request to be removed, though I do agree it should go

File: modular_skyrat/master_files/code/_globalvars/maint_loot_common.dm [hunk 3]
```diff
@@ -111,7 +112,8 @@ GLOBAL_LIST_INIT(common_loot, list( //common: basic items
 		/obj/item/storage/firstaid/medical = 25,
 		/obj/item/storage/firstaid/o2 = 25,
 		/obj/item/storage/firstaid/regular = 25,
-		/obj/item/storage/firstaid/toxin = 25
+		/obj/item/storage/firstaid/toxin = 25,
+		/obj/item/lead_pipe = 25
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/_globalvars/maint_loot_common.dm]:
```suggestion
		/obj/item/lead_pipe = 25,
```

File: modular_skyrat/master_files/code/_globalvars/maint_loot_common.dm [hunk 4]
```diff
@@ -353,6 +355,7 @@ GLOBAL_LIST_INIT(common_loot, list( //common: basic items
 		/obj/item/food/meat/slab/human/mutant/shadow = 1,
 		/obj/item/food/meat/slab/human/mutant/skeleton = 1,
 		/obj/item/food/meat/slab/human/mutant/slime = 1,
-		/obj/item/food/meat/slab/human/mutant/zombie = 1
+		/obj/item/food/meat/slab/human/mutant/zombie = 1,
+		/obj/item/seeds/kronkus = 1
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/_globalvars/maint_loot_common.dm]:
```suggestion
		/obj/item/seeds/kronkus = 1,
```

File: modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm [hunk 5]
```diff
@@ -65,5 +58,6 @@ GLOBAL_LIST_INIT(oddity_loot, list(//oddity: strange or crazy items
 	/obj/item/storage/toolbox/syndicate = 100,
 	/obj/item/storage/toolbox/infiltrator = 100,
 	/obj/item/toy/sprayoncan = 200,
-	/obj/item/warpwhistle = 5
+	/obj/item/warpwhistle = 5,
+	/obj/item/shadowcloak = 50
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm]: Also, what does the shadowcloak do, again?
```suggestion
	/obj/item/shadowcloak = 50,
```

File: modular_skyrat/master_files/code/_globalvars/maint_loot_uncommon.dm [hunk 6]
```diff
@@ -319,5 +319,9 @@ GLOBAL_LIST_INIT(uncommon_loot, list(//uncommon: useful items
 		/obj/item/eldritch_potion/crucible_soul = 1,
 		/obj/item/eldritch_potion/duskndawn = 1,
 		/obj/item/eldritch_potion/wounded = 1
-	) = 1
+	) = 1,
+	list( //Modsuits
+		/obj/item/mod/construction/broken_core = 1,
+		/obj/effect/spawner/random/mod/maint = 3
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/_globalvars/maint_loot_uncommon.dm]:
```suggestion
		/obj/effect/spawner/random/mod/maint = 3,
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/master_files/code/_globalvars/maint_loot_uncommon.dm (hunk 6 above)
Comment (RatFromTheJungle) [reply modular_skyrat/master_files/code/_globalvars/maint_loot_uncommon.dm]: the code does not like it if you put a comma on the last item in a list

File: modular_skyrat/master_files/code/_globalvars/maint_loot_common.dm (hunk 4 above)
Comment (RatFromTheJungle) [reply modular_skyrat/master_files/code/_globalvars/maint_loot_common.dm]: the code does not like it if you put a comma on the last item in a list

File: modular_skyrat/master_files/code/_globalvars/maint_loot_common.dm (hunk 3 above)
Comment (RatFromTheJungle) [reply modular_skyrat/master_files/code/_globalvars/maint_loot_common.dm]: the code does not like it if you put a comma on the last item in a list

File: modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm (hunk 5 above)
Comment (RatFromTheJungle) [reply modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm]: the code does not like it if you put a comma on the last item in a list

Also, it lets you cloak to like, modsuit tiers, but can only recharge in absolute darkness, and you get like, 4s of invisiblity, it's a tf2 spy thing

File: modular_skyrat/master_files/code/_globalvars/maint_loot_common.dm (hunk 3 above)
Comment (GoldenAlpharex) [reply modular_skyrat/master_files/code/_globalvars/maint_loot_common.dm]: > the code does not like it if you put a comma on the last item in a list

Not true, look at many other files we have that does that. Check [here](https://github.com/Skyrat-SS13/Skyrat-tg/blob/36654a220a1588864d30c05377c07423bcc46352/modular_skyrat/modules/modular_vending/code/autodrobe.dm#L64) for an example.

File: modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm [hunk 7]
```diff
@@ -17,10 +17,7 @@ GLOBAL_LIST_INIT(oddity_loot, list(//oddity: strange or crazy items
 	/obj/item/clothing/mask/animal/frog/cursed = 50,
 	/obj/item/clothing/mask/animal/horsehead/cursed = 50,
 	/obj/item/clothing/mask/animal/pig/cursed = 50,
-	/obj/item/clothing/mask/void_mask = 5,
 	/obj/item/clothing/shoes/bhop = 5,
-	/obj/item/clothing/shoes/clown_shoes/banana_shoes/combat = 5,
```
Comment (MassacreDemigirl) [inline modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm:22]: A desyncronizer is ok but this is not?

Comment (tf-4) [reply modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm:22]: yes

**Comment by Fikou** [comment]
why did you remove the brainwashing skillchip

File: modular_skyrat/master_files/code/_globalvars/maint_loot_uncommon.dm [hunk 8]
```diff
@@ -203,7 +203,6 @@ GLOBAL_LIST_INIT(uncommon_loot, list(//uncommon: useful items
 		/obj/item/rsf = 5,
 		/obj/item/rupee = 5,
 		/obj/item/sord = 5,
-		/obj/item/stack/telecrystal/five = 5,
 		/obj/item/stack/telecrystal = 5,
```
Comment (Iamgoofball) [inline modular_skyrat/master_files/code/_globalvars/maint_loot_uncommon.dm:206]: IM SORRY WHAT

Comment (Iamgoofball) [reply modular_skyrat/master_files/code/_globalvars/maint_loot_uncommon.dm:206]: no, fucking, no, no telecrystals in maint loot, bad, awful, bad, remove this

**Pull request merged**